pom.xml

```diff
@@ -99,6 +99,7 @@
     
     
     <properties>
+        <caf.worker-document.version>3.0.0-SNAPSHOT</caf.worker-document.version>
```

Comment:
There's something odd here.  The Worker Framework version should be implied by the Document Worker Framework version - you shouldn't have to specify both.  DWF 3.0.0 uses WF 2.0.0, but only the DWF version should appear in this file.